nilearn/plotting/__init__.py

```diff
@@ -46,8 +46,10 @@ def _set_mpl_backend():
     plot_markers, plot_prob_atlas, plot_carpet, plot_img_comparison, show)
 from .find_cuts import find_xyz_cut_coords, find_cut_slices, \
     find_parcellation_cut_coords, find_probabilistic_atlas_cut_coords
-from .matrix_plotting import (plot_matrix, plot_contrast_matrix,
-                              plot_design_matrix, plot_event)
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
```

Comment:
Can we be a bit more specific about which warnings we ignore, so as to avoid silencing legit warnings:
- Could we move these lines into the specific file (matrix_plotting)
- Could we can only FutureWarnings